packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -315,6 +318,30 @@ bool _checkXcodeVersion() {
   return true;
 }
 
+bool _checkCocoaPodsInstalled() {
+  if (!platform.isMacOS)
+    return false;
+  return exitsHappy(<String>['pod', '--version']);
+}
+
+void _installCocoaPods(Directory bundle, String engineDirectory)  {
```

Comment:
[nit] Ambiguous name: are we installing the cocoapods tool or a particular cocoa pod?